ref-exchange/Cargo.toml

```diff
@@ -9,10 +9,10 @@ publish = false
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-near-sdk = { git = "https://github.com/near/near-sdk-rs", rev = "9d99077" }
-near-contract-standards = { git = "https://github.com/near/near-sdk-rs", rev = "9d99077" }
 uint = { version = "0.9.0", default-features = false }
+near-sdk = "3.1.0"
```

Comment:
also there is 4.0 of sdk now, isn't it?
There is PR for this in #34

Reply:
I think `4.0.0` is not published yet. Only the preview version. Better wait some time before upgrading.

Reply:
OK, I also prefer to keeping 3.1.0 for safety.